cmsEditArticle.php

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+
+<?php
+
+require_once 'DbConnect.php';
+require_once 'dropdownDB.php';
+require_once 'getArticleImgDB.php';
+require_once 'updateArticleDB.php';
+
+$db = connectToDB();
+
+$dropdownKey = 'title';
+$dropdownArray = getArticleTitles($db, $dropdownKey);
+$titleDropdown = createDropdownfromDB($dropdownArray, $dropdownKey);
+
+$actionType = $_POST['updateArticle'];
```

Comment:
all these variables are useless, just use $_POST. all your doing is duplicating the data